<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MineSweepingClass</class>
 <widget class="QMainWindow" name="MineSweepingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>243</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MineSweeping</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">/*QWidget#centralWidget
{
	background-color: rgb(217, 226, 235);
}*/</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <item row="2" column="3">
     <widget class="QLabel" name="labelTime">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>0</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:16px;
font: 75 9pt &quot;微软雅黑&quot;;
background-color:rgb(150,150,150);
color:rgb(244,244,244);
border-radius:5px</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QLabel" name="bombIcon">
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLabel" name="labelBombCount">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>0</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:16px;
font: 75 9pt &quot;微软雅黑&quot;;
background-color:rgb(150,150,150);
color:rgb(244,244,244);
border-radius:5px</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="timeIcon">
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="8">
     <widget class="ITableWidget" name="tableWidget">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>10</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>10</number>
      </attribute>
     </widget>
    </item>
    <item row="2" column="7">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="4">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>192</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>游戏(&amp;G)</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionInfo"/>
    <addaction name="actionSetting"/>
    <addaction name="actionStyle"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="actionHelp">
   <property name="text">
    <string>查看帮助</string>
   </property>
   <property name="statusTip">
    <string>查看扫雷帮助文档(从Window提取)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于扫雷</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>是否</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>统计信息</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>统计信息</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="text">
    <string>选项</string>
   </property>
  </action>
  <action name="actionStyle">
   <property name="text">
    <string>更改外观</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>新游戏</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ITableWidget</class>
   <extends>QTableWidget</extends>
   <header>itablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="minesweeping.qrc"/>
 </resources>
 <connections/>
</ui>
